xml_inscriptions/transcribed/USEP Penn Museum Amphora Handle.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remember that the file name should match the USEP ID, so that will need to be changed.